magenta/models/shared/melody_rnn_generate.py

```diff
@@ -198,7 +202,7 @@ def run(melody_encoder_decoder, build_graph):
                       FLAGS.primer_midi)
 
   run_generate(graph, train_dir, FLAGS.output_dir, melody_encoder_decoder,
-               primer_melody, FLAGS.num_steps, bpm)
+               primer_melody, FLAGS.num_steps, FLAGS.bpm)
```

Comment:
Your right @cghawthorne. I would like the bpm to replace the primer if it is mentioned in the command line, however my code always ignores the primer bpm. I wil see what I can do to change it.